src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,24 @@
+.cards {
+
+  &__row {
```

Comment:
You need to define `max-width` according to task
<img width="1022" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/49764881/47bc6b04-c8e6-4344-969f-03d18bc88360">